### ISSUE TITLE
flatpak install errors when executed from inside a chroot

### ISSUE THREAD

**Issue by LaurentBonnaud** [flatpak/flatpak#703]
Hi,

I am trying to use flatpak to install software inside a Debian stretch chroot with this package version:
```
Package: flatpak
Version: 0.8.5-1
```
and I saw 2 error messages:

1.  `X11 connection rejected because of wrong authentication.`
Why is flatpak trying to access my X server ?

2. `Failed to make / slave: Invalid argument`

The installation appears to have completed successfully.

Here is a complete install log:
```
# flatpak install --from http://download.documentfoundation.org/libreoffice/flatpak/LibreOffice.flatpakref
X11 connection rejected because of wrong authentication.
X11 connection rejected because of wrong authentication.
Installing: org.libreoffice.LibreOffice/x86_64/fresh
Updating: org.gnome.Platform/x86_64/3.20 from gnome
No updates.
Updating: org.gnome.Platform.Locale/x86_64/3.20 from gnome
No updates.
Installing: org.libreoffice.LibreOffice/x86_64/fresh from org.libreoffice.LibreOffice-4-origin

13 delta parts, 99 loose fetched; 233625 KiB transferred in 17 seconds
Failed to make / slave: Invalid argument
Failed to make / slave: Invalid argument
Failed to make / slave: Invalid argument
```

**Comment by alexlarsson** [comment]
Yeah, this is a bubblewrap issue.
You can work around it by making a bind mount at the root of the chroot i believe.

**Comment by alexlarsson** [comment]
Unclear why it tried to connect to X though. Flatpak doesn't even link to libX11.

**Comment by alexlarsson** [comment]
i.e. `mount --bind /the/chroot /the/chroot`

**Comment by matthiasclasen** [comment]
Closing this, since it was reported against an old version. Please reopen, if the issue still exists with flatpak 1.0.x.

**Comment by tiibz64** [comment]
Hi,
i'm trying to install Linphone with flatpak inside a chroot environnement and i have the same error mesage about bubblewrap:
bwrap: Failed to make / slave: Invalid argument
bwrap: Failed to make / slave: Invalid argument
bwrap: Failed to make / slave: Invalid argument

Linphone seems to be correctly installed and the generated image works on some PCs but trying to install the same image in a VM results in a black screen after install.
I tried only installing flatpak via apt then linphone and nothing else to make sure to avoid conflicts with other software but the error is still there and my VM can't boot.

I already tried the instructions posted earlier ("i.e. mount --bind /the/chroot /the/chroot") but the result is the same.

I'm using a Xubuntu 18.04.2 image and i'm using CUBIC to customize it, host OS is Ubuntu 16.04.

Any help would be greatly appreciated.

Thanks
